stob/ad.h

```diff
@@ -107,6 +107,13 @@ struct m0_ad_balloc_ops {
 };
 
 enum { AD_PATHLEN = 4096 };
+// Defines hash table size for struct m0_be_emap. This size should be pow of 2,
+// and maximum allowed size is 65536.
+#define EMAP_HT_SIZE 256
```

Comment:
1. Don't use C++ style comments.
2. Use "enum" instead of "define".

Reply:
Handled in commit id 5cfda07